src/corefx/System.Globalization.Native/CMakeLists.txt

```diff
@@ -9,16 +9,25 @@ add_definitions(-DBIT64=1)
 set(ICU_HOMEBREW_LIB_PATH "/usr/local/opt/icu4c/lib")
 set(ICU_HOMEBREW_INC_PATH "/usr/local/opt/icu4c/include")
 
-find_library(ICUUC NAMES icuuc PATHS ${ICU_HOMEBREW_LIB_PATH})
-if(ICUUC STREQUAL ICUUC-NOTFOUND)
-    message(FATAL_ERROR "Cannot find libicuuc, try installing libicu-dev (or the appropriate package for your platform)")
-    return()
-endif()
-
-find_library(ICUI18N NAMES icui18n PATHS ${ICU_HOMEBREW_LIB_PATH})
-if(ICUI18N STREQUAL ICUI18N-NOTFOUND)
-    message(FATAL_ERROR "Cannot find libicui18n, try installing libicu-dev (or the appropriate package for your platform)")
-    return()
+if(NOT CLR_CMAKE_PLATFORM_DARWIN)
+    find_library(ICUUC NAMES icuuc PATHS ${ICU_HOMEBREW_LIB_PATH})
```

Comment:
How do you feel about switching this to use find_package(ICU REQUIRED), and then use the ${ICU_LIBRARIES} in target_link_libraries?